toqito/matrix_props/is_totally_positive.py

```diff
@@ -40,6 +40,42 @@ def is_totally_positive(mat: np.ndarray, tol: float = 1e-6, sub_sizes: list | No
     .. math::
         \text{det}(X) = 1 \times 4 - 2 \times 3 = 4 - 6 = 2
 
+    Our function indicates that this matrix is indeed totally positive.
```

Comment:
@juancaCeb Not somethign you made changes to but can you please add $X = $ on line 19?